src/screens/Home/components/SideContent/index.js

```diff
@@ -0,0 +1,145 @@
+import React, { useState } from 'react';
+import { useQuery } from '@apollo/client';
+import { string } from 'prop-types';
+import i18next from 'i18next';
+
+import Loading from '@components/Loading';
+import { currencyFormat } from '@constants/utils';
+import { COUNTRY_QUERY } from '../../constants';
+import styles from './styles.module.scss';
+
+function SideContent({ id }) {
+  const { loading } = useQuery(COUNTRY_QUERY, {
+    variables: { id },
+    onCompleted: (data) => setCountry(data.Country[0]),
+  });
+  const [country, setCountry] = useState();
+
+  return loading || !country ? (
+    <Loading isSmall />
+  ) : (
+    <div className={styles.content}>
+      <img
+        className={`m-bottom-3 ${styles.img}`}
+        src={country.flag.svgFile}
+        alt={country.name}
+      />
+      <h2 id='dialog-title' className='small-title m-bottom-1'>
+        {country.name}
+      </h2>
+      <h3 className='big-text fw-thin m-bottom-3'>
+        <span
+          className='base-text m-right-1'
+          role='img'
+          aria-label={country.name}
+        >
+          {country.flag.emoji}
+        </span>
+        {country.nativeName} ({country.alpha3Code})
+      </h3>
+      <div className={`m-bottom-1 ${styles.innerContent}`}>
+        <div className={`m-bottom-2 m-right-1 ${styles.group}`}>
+          <span className='small-text m-bottom-1'>
+            {i18next.t('HOME:COUNTRY_AREA')}
+          </span>
+          <span className='base-text'>{country.area}</span>
+        </div>
+        <div className={`m-bottom-2 m-right-1 ${styles.group}`}>
+          <span className='small-text m-bottom-1'>
+            {i18next.t('HOME:COUNTRY_POPULATION')}
+          </span>
+          <span className='base-text'>
+            {currencyFormat(country.population)}
+          </span>
+        </div>
+        <div className={`m-bottom-2 ${styles.group}`}>
```

Comment:
I think there's a repeated pattern here which could be a separate component you could call:

```
function ChooseTheName({ prop1, prop2 }) {
  return (
        <div className={`m-bottom-2 m-right-1 ${styles.group}`}>
          <span className='small-text m-bottom-1'>
            {prop1}
          </span>
          <span className='base-text'>
            {prop2}
          </span>
        </div>
  )
}
```